src/App.vue

```diff
@@ -41,12 +43,24 @@ export default defineComponent({
   components: { TimeZoneCard, AddTimeZoneModal },
 
   setup() {
+    const preferences = ref({
```

Comment:
Preferences would be nice to store in [localStorage](https://developer.mozilla.org/en-US/docs/Web/API/Window/localStorage) like time zones list.

Please, make a [watcher](https://v3.vuejs.org/api/computed-watch-api.html#watch) and if preferences updated, write them into localStorage using [JSON.parse](https://www.digitalocean.com/community/tutorials/js-json-parse-stringify) and by default, get preferences object from localStorage or create new object. 

If you make that, preferences will be saved in the user's browser.